### PR TITLE
Reword ELI5 prompt in microsimulation ai analysis

### LINKED ISSUE #1867: Claude refuses to produce analysis for ELI5 prompt

### PR DESCRIPTION
**Pull request by tawandamoyo**
Fixes #1867

### CODE REVIEW

**Comment by anth-volk** [comment]
@tawandamoyo Thanks for this. It looks like a pinned version of `markupsafe` is causing this to fail. Could you open a new PR, remove the pinned version from the `setup.py`, attempt to install, and then open a PR and request review on that? It should be quick, and until that's done, I think we'll probably run into this consistently.

**Comment by MaxGhenis** [comment]
I relaxed markupsafe in #1871. Updating here and will merge.
